src/Grid/Grid.levers.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Grid gap or not
+ */
+export const gridGap = [true, false];
+
+/**
+ * Available number of grid columns at XS breakpoint (1 to 12)
+ */
+export const gridColsXs = Array.from({length: 12}, (_, i) => i + 1);
```

Comment:
Grid cols will always be the same for each breakpoint, won't they? If yes, each of these breakpoint col variables could be consolidated down into one.

Reply:
Originally I played with having just 1-6 for XS but I changed my mind 🤔

Reply:
All done